### PR TITLE
Update disk.json to add burstingEnabled flag

### PR DESCRIPTION
**Pull request by rahul602**
Support to enable disable disk bursting feature would allow customers to burst beyond the provisioned perf target up to 30K IOPS.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json [hunk 1]
```diff
@@ -1647,6 +1647,10 @@
         "tier": {
           "type": "string",
           "description": "Performance tier of the disk (e.g, P4, S10) as described here: https://azure.microsoft.com/en-us/pricing/details/managed-disks/. Does not apply to Ultra disks."
+        },
+        "burstingEnabled": {
+          "type": "boolean",
+          "description": "Disk Bursting property can be used to enable support for burst beyond the provisioned perf target, up to 30K IOPS. Bursting is disabled by default. Does not apply to Ultra disks."
```
Comment (vikramd-ms) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json]: "Set to true to enable bursting beyond the provisioned performance target of the disk."

Avoid mentioning numbers like 30 K which can change. It is better to add an aka.ms link to the paid bursting documentation when it is available.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json [hunk 2]
```diff
@@ -1647,6 +1647,10 @@
         "tier": {
           "type": "string",
           "description": "Performance tier of the disk (e.g, P4, S10) as described here: https://azure.microsoft.com/en-us/pricing/details/managed-disks/. Does not apply to Ultra disks."
+        },
+        "burstingEnabled": {
```
Comment (vikramd-ms) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json:1651]: Add example.

Comment (rahul602) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json:1651]: Added example for enable and disable bursting

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json [hunk 3]
```diff
@@ -1647,6 +1647,10 @@
...
+          "type": "boolean",
+          "description": "Set to true to enable bursting beyond the provisioned performance target of the disk. Bursting is disabled by default. Does not apply to Ultra disks."
```
Comment (vikramd-ms) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json:1653]: Isn't burstig only allowed for Premium disks and not Standard disks? If so, please say so.

**Review by vikramd-ms — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json (hunk 3 above)
Comment (rahul602) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json:1653]: I think we should not mention that in the swagger. There is plan to enable bursting for Standard SSD as well.

**Pull request merged**